internal/cli/login.go

```diff
@@ -317,7 +317,7 @@ func (o *LoginOptions) Run(ctx context.Context, args []string) error {
 			fmt.Printf("You must obtain an API token by visiting %s/request\n", oauthConfig.TokenEndpoint)
 			fmt.Printf("Then login via \"flightctl login %s --token=<token>\"\n", config.Service.Server)
 			fmt.Printf("Alternatively, use \"flightctl login %s --web\" to login via your browser\n", config.Service.Server)
-			return nil
+			return fmt.Errorf("no token provided")
```

Comment:
This was necessary to detect when a follow-up on the login command was necessary because it did not succeed.